nuimo/src/main/kotlin/com/senic/nuimo/NuimoBluetoothController.kt

```diff
@@ -26,6 +26,9 @@ class NuimoBluetoothController(bluetoothDevice: BluetoothDevice, context: Contex
     private val mainHandler = Handler(Looper.getMainLooper())
     private var writeQueue = WriteQueue()
     private var matrixWriter: LedMatrixWriter? = null
+    private var rebootToDfuModeCharacteristic: BluetoothGattCharacteristic? = null
+    var supportsRebootToDfuMode = false
```

Comment:
With previous comment, this can then also be computed with `rebootToDfuModeCharacteristic != nil`